src/tools/perf/perftest.c

```diff
@@ -1075,13 +1075,16 @@ static ucs_status_t setup_sock_rte(struct perftest_context *ctx)
 
         close(sockfd);
 
+        free(ctx->params.super.msg_size_list);
```

Comment:
where was this allocated before? can you pls add comment to explain why need to free here?

Reply:
it was allocated in `init_test_params()` for server and client sides
added the comment